mbuild/utils/decorators.py

```diff
@@ -1,7 +1,23 @@
 """Some helpful decorators."""
+from functools import wraps
+from warnings import warn
 
 
-from warnings import warn
+def make_callable(decorator):
```

Comment:
I think we could safely add pragma no cover comments to all these decorators.

Reply:
On board with this.